server/raft.go

```diff
@@ -1528,6 +1543,12 @@ func (n *raft) resetElectionTimeout() {
 	n.resetElect(randElectionTimeout())
 }
 
+func (n *raft) resetElectionTimeoutWithLock() {
```

Comment:
Just have this one call resetElectWithLock(randElectionTimeout())?

Reply:
done

Reply:
done